src/main/java/org/broadinstitute/hellbender/utils/read/GATKRead.java

```diff
@@ -217,6 +217,17 @@ default String getBasesString() {
     void setBaseQualities( final byte[] baseQualities );
 
     /**
+     * @return Base qualities string as printable ASCII values.
+     */
+    String getBaseQualityString();
+
+    /**
+     * Set the base qualities from a string of ASCII encoded values
+     * @param baseQualities - ASCII encode values of base qualities.
+     */
+    void setBaseQualityString( final String baseQualities );
```

Comment:
@droazen In the past there has been resistance to exposing string methods for base qualities.  How do we you feel about having these?